pyproject.toml

```diff
@@ -17,19 +17,17 @@ maintainers = [
     {name = 'Waylan Limberg', email = 'python.markdown@gmail.com'},
     {name = 'Isaac Muse'}
 ]
-license = {file = 'LICENSE.md'}
-requires-python = '>=3.8'
+license = "BSD-3-Clause"
```

Comment:
Shouldn't this be `License-Expression = "BSD-3-Clause"`

Reply:
No. `License-Expression` is a field in the generated `PKG-INFO` file. In `pyproject.toml` it is just `license`: <https://packaging.python.org/en/latest/specifications/pyproject-toml/#license>.

Reply:
That's not confusing at all.